examples/dbscan/ArborX_DBSCAN.hpp

```diff
@@ -245,10 +265,10 @@ void dbscan(ExecutionSpace exec_space, Primitives const &primitives,
                                                   1);
                        });
 
-  // The idea here is to replace cluster indices for small clusters (containing
-  // less than cluster_min_size points) to INT_MAX. This way, during the sort
-  // routine afterwards, they will be at the end of the permutation array,
-  // allowing us to simply truncate it to get the result.
+  // The idea here is to replace cluster indices for small clusters
+  // (containing less than cluster_min_size points) to INT_MAX. This way,
+  // during the sort routine afterwards, they will be at the end of the
+  // permutation array, allowing us to simply truncate it to get the result.
```

Comment:
Are the changes to the formatting here intentional?

Reply:
No, just saved the file and it auto-formatted. Will revert.